custom/enikshay/const.py

```diff
@@ -184,3 +184,5 @@
 }
 
 LOCATION_SITE_CODE_MEHSANA = 'dto_gu_msn'
+FIXTURE_FOR_EPISODE_LEDGERS = "adherence_ledger_values"
```

Comment:
nit: would be nice to keep these 2 names grammatically consistent, so this one would be `EPISODE_LEDGER_FIXTURE`